### PR TITLE
Ignore

### PR DESCRIPTION
**Pull request by bombcar**
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9271 by ignoring region files with () in it.

Likely there's a better way that can be refactored in but this works for now.

Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9128

### CODE REVIEW

**Comment by bombcar** [comment]
Now with 900% more regex. having to escape `\` hurts

**Comment by bombcar** [comment]
```
[20:26:37] [Server thread/INFO]: [VisualProspecting] Processing dimension with id 112 with fast scanning.
[20:26:37] [ForkJoinPool.commonPool-worker-0/WARN]: [VisualProspecting] Invalid region file found! r.-2.2(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-7/WARN]: [VisualProspecting] Invalid region file found! r.1.0(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-3/WARN]: [VisualProspecting] Invalid region file found! r.-2.1(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-1/WARN]: [VisualProspecting] Invalid region file found! r.0.0(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-1/WARN]: [VisualProspecting] Invalid region file found! r.-1.2(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-0/WARN]: [VisualProspecting] Invalid region file found! r.-1.1(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-3/WARN]: [VisualProspecting] Invalid region file found! r.-3.0(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-3/WARN]: [VisualProspecting] Invalid region file found! r.-2.0(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-3/WARN]: [VisualProspecting] Invalid region file found! r.-1.3(1).mca continuing
[20:26:37] [ForkJoinPool.commonPool-worker-3/WARN]: [VisualProspecting] Invalid region file found! r.-1.0(1).mca continuing
[20:26:37] [Server thread/INFO]: [VisualProspecting] Processing dimension with id 85 with fast scanning.
[20:26:37] [Server thread/INFO]: [VisualProspecting] Processing dimension with id 49 with fast scanning.
[20:26:37] [Server thread/INFO]: [VisualProspecting] Processing dimension with id 82 with fast scanning.
[20:26:37] [Server thread/INFO]: [VisualProspecting] Processing dimension with id -1 with slow (safe) scanning.
```

File: src/main/java/com/sinthoras/visualprospecting/database/cachebuilder/DimensionAnalysis.java [hunk 1]
```diff
@@ -90,6 +92,10 @@ public void processMinecraftWorld(MinecraftWorld world) throws IOException {
 
     private void executeForEachGeneratedOreChunk(File regionFile, IChunkHandler chunkHandler) {
         try {
+            if ( !Pattern.matches("^r\\.-?\\d+\\.-?\\d+.mca$", regionFile.getName())) {
+                VP.warn("Invalid region file found! " + regionFile.getName() + " continuing");
```
Comment (SinTh0r4s) [inline src/main/java/com/sinthoras/visualprospecting/database/cachebuilder/DimensionAnalysis.java:96]: Dies that print the full path? It is important to get the dimension Id  to the command line as well

**Review by SinTh0r4s — APPROVED** [review]

File: src/main/java/com/sinthoras/visualprospecting/database/cachebuilder/DimensionAnalysis.java (hunk 1 above)
Comment (bombcar) [reply src/main/java/com/sinthoras/visualprospecting/database/cachebuilder/DimensionAnalysis.java:96]: It’s just the malformed file, but we could dump the whole path. 

Usually it’s a “backup” of a region file made by some restore tool.

Comment (SinTh0r4s) [reply src/main/java/com/sinthoras/visualprospecting/database/cachebuilder/DimensionAnalysis.java:96]: Yes, I know. Just that the user knows Wich files creates the trouble and can inspect it manually. He needs to know the dimension for it so the full path+file name would be great. That's all ;-)

Comment (bombcar) [reply src/main/java/com/sinthoras/visualprospecting/database/cachebuilder/DimensionAnalysis.java:96]: Do we have that easily available somehow? Firing up the IDE but like regionFile.getPath()?

Comment (bombcar) [reply src/main/java/com/sinthoras/visualprospecting/database/cachebuilder/DimensionAnalysis.java:96]: hmmm .getCanonicalPath should work

**Pull request merged**